atd-vzd/views/view_crashes_mainlane_noncr3.sql

```diff
@@ -0,0 +1,11 @@
+CREATE OR REPLACE VIEW view_crashes_mainlane_noncr3 AS
```

Comment:
are we using these two mainline views in the UI? or are they for Franks and reporting? just curious.

Reply:
They are not currently used or tracked in Hasura, but I created them to document how Frank finds main-lane crashes.

Reply:
nice, thanks!